pkg/render/apiserver.go

```diff
@@ -801,6 +805,11 @@ func (c *apiServerComponent) apiServerDeployment() *appsv1.Deployment {
 			rmeta.APIServerNamespace(c.installation.Variant)))
 	}
 
+	var replicas int32 = DefaultReplicas
```

Comment:
We generally do defaulting in the controller, not in the render package. I think this should be no different, unless you have an explicit reason for not wanting to follow the convention?